### DISCUSSION TITLE
Forgalmazás engedélyezése

### DISCUSSION THREAD

**Discussion by wlln22** [nav-gov-hu/eRECEIPT#35] [Kérdések és válaszok]
A jelenlegi rendelet szerint az ügyintézés határideje 15 + 15 nap. Az epg forgalmi engedélyezésének mi lesz a határideje?

"Felhőalapú e-pénztárgép forgalmazásának engedélyezés iránti kérelem további tartalma: azon alkalmazás megjelölése, amelyet használva hordozható e-pénztárgépként működik." - A felhőalapú epg kötelezően hordozható. Itt egy további alkalmazásra utal a szöveg, vagy magára az epg alkalmazásra?

**Comment by BatariZ** [comment]
Az időközben megjelent rendelet (A nemzetgazdasági miniszter 8/2025. (III. 31.) NGM rendelete az e-pénztárgépek forgalmazásáról, üzemeltetéséről, valamint az e-pénztárgépek és az e-nyugta kiállításának követelményeiről), illetve az aktualizált Fejlesztői dokumentáció tartalmazza a válaszokat a feltett kérdésekre.
